rust/private/rustc.bzl

```diff
@@ -866,7 +866,7 @@ def _crate_to_link_flag(crate_info):
     Returns:
         list: Link flags for the current crate info
     """
-    return ["--extern", "{}={}".format(crate_info.name, crate_info.dep.output.path)]
+    return ["--extern={}={}".format(crate_info.name, crate_info.dep.output.path)]
```

Comment:
Nit: this is a change in behavior, and the PR claims it's only adding a test. I understand that this is done to make testing easier, but let's document that at least in the PR description.

Btw, I'd be happy to accept a PR that normalizes all flags we pass to rustc to use consistent format :)